db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql

```diff
@@ -7,7 +7,7 @@
 -- 
 -- Modified Date        Modified By         Comments / Remarks
 -- -------------------------------------------------------------------------------------------------
--- Apr-2021		Ram Bhatt	   Added resume_remove_tags column to registration table
+-- Apr-2021		Ram Bhatt	   Added pause_rule_ids column to registration table
```

Comment:
Modified date is not matching with the activity, I think you have to add it as 2 activities at the end that resume remove is removed and puase rule ids added